x-pack/plugins/cases/public/components/case_view/components/case_view_activity.tsx

```diff
@@ -180,6 +188,11 @@ export const CaseViewActivity = ({
         )}
       </EuiFlexItem>
       <EuiFlexItem grow={2}>
+        <SeveritySidebarSelector
```

Comment:
Could you please disable the dropdown if the user does not have write access (`userCanCrud`)?

Reply:
I totally missed that part. I just added it.

Reply:
Thanks!